arm_pose:
  home :
  # joint name : angle(degree)
    joint1 : 0.0
    joint2 : -60
    joint3 : 20
    joint4 : 40
    gripper : 0.745 
    
  home_with_object :
    joint1 : 0.0
    joint2 : -70 
    joint3 : 50 
    joint4 : 20 
      
  via_pose : 
    joint1 : 0.0
    joint2 : -18
    joint3 : 21
    joint4 : -3
    
  termination_pose :
    joint1 : 0.0
    joint2 : 0.0
    joint3 : 20.0
    joint4 : 70.0
    
  target_pose :
    joint1 : 0.0
    joint2 : -15.0
    joint3 : 31.5
    joint4 : -9.0
  
approach:
  distance : 0.075
  linear_vel : 0.05
  angular_vel : 0.1
  interval_sleep_ms : 500
  repeat_times : 2
  
leave : 
  linear_vel : 0.15
  angular_vel : 0.5
  interval_sleep_ms : 500
